mautic/mautic
# Email sent date filter on segments

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | Yes
| Automated tests included? | 
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description: 
Added Filter option to segments to find out when a contact was last sent an email. Same code as for when a contact last read an email; only change is the column (date_sent over date_read) the query looks at in the email_stats table.

[//]: # ( As applicable: )
#### Steps to reproduce the bug:
1. 
2. 

#### Steps to test this PR:
1.  Send an email to a user
2.  Create a segment with a filter looking at "Email Sent Date" to be greater than day sent the email. 
3.  Rebuild segments by running php app/console mautic:segments:update
4. Contact should be in the segment. 

#### List deprecations along with the new alternative:
1.  N/A
2. 

#### List backwards compatibility breaks:
1. 
2.
